langchain/src/document_loaders/web/wikipedia.ts

```diff
@@ -0,0 +1,131 @@
+import wiki, { Page } from "wikipedia";
+import { Document } from "../../document.js";
+import { BaseDocumentLoader } from "../base.js";
+
+const DISCARDCATEGORIES = [
```

Comment:
I think these should be moved to examples instead, as this assumes only English locale will be loaded.